src/components/ConsoleInput.jsx

```diff
@@ -51,15 +73,25 @@ export default class ConsoleInput extends Component {
   }
 
   render() {
-    return <div className="console__input" ref={this._ref} />;
+    return (
+      <div
+        className="console__input"
+        ref={this._ref}
+        // eslint-disable-next-line react/jsx-no-bind
+        onClick={e => e.stopPropagation()}
```

Comment:
I would just use `react-prevent-clickthrough` here (arguably we should have used it in the other one too but more of a slam dunk when the _only_ thing we’re doing is preventing clickthrough)